src/main/java/uk/gov/hmcts/sptribs/judicialrefdata/JudicialClient.java

```diff
@@ -25,6 +25,7 @@ public interface JudicialClient {
     List<UserProfileRefreshResponse> getUserProfiles(
         @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorisation,
         @RequestHeader(AUTHORIZATION) final String authorisation,
+        @RequestHeader(ACCEPT) final String accept,
```

Comment:
Does it not need change in JudicialService class as this adding new parameter to method?